shows/left_to_right_and_back.py

```diff
@@ -1,27 +1,51 @@
-from color import RGB
+from color import HSV
 from .showbase import ShowBase
 from grid import Grid, Direction, sweep
-
+from grid.cell import Direction, Position, row_length
+from grid import traversal
+import time
 
 class LeftToRightAndBack(ShowBase):
     def __init__(self, grid: Grid, frame_delay: float = 1.0):
         self.grid = grid
         self.frame_delay = frame_delay
 
     def next_frame(self):
-        fwd = True
+        n_rows = self.grid.row_count
+        hsv = HSV(0.0,0.9,.5)
 
-        while True:
-            direction = Direction.LEFT_TO_RIGHT if fwd else Direction.RIGHT_TO_LEFT
-            sequence = sweep(direction, self.grid.row_count)
+        pix_arr = []
+        a_ctr = 0
+        for points in traversal.left_to_right(n_rows):
+            for (row, col) in points:
+                cell = self.grid.select(Position(row=row,col=col))
```

Comment:
This can be `cell = self.grid[Position(row, col)]` and you don't have to do `cell[0]` after.